manage_deps/workflows.rst

```diff
@@ -39,16 +39,16 @@ So the layout will be:
       main.cpp
 
 
-Now you are ready for building:
+Now you are ready to build:
 
 .. code-block:: bash
    
       $ cmake ../example-hello -G ...
       $ cmake --build . --config Release
       
-With this configuration you are typically working in just one configuration of the project, 
-that will be destroyed and create a new build with a new configuration with different settings
-if needed.
+In effect, with this setup, we have created a separate configuration of the project, without
+affecting its default one. The benefit is that we can experiment freely, and even erase it and
```

Comment:
Not the idea, there is not a default one, so we are not creating a separate configuration. We are just building one configuration, if you want to build a different one, just remove build and temp files & restart with different conan install parameters